spec/models/invoice_spec.rb

```diff
@@ -21,17 +21,33 @@
     expect(Invoice.all.length).to eq(2)
   end
 
-  # before :each do
-  #
-  # end
   #
   # describe 'class methods' do
   #   describe '.' do
   #   end
   # end
   #
```

Comment:
We can delete the unused comments / commented out code ... unless we think we'll need it later.  But I'd vote delete it.

Reply:
I vote that we can clean things up after the merge